### DISCUSSION TITLE
Labels - Custom sort order for labels

### DISCUSSION THREAD

**Discussion by Levi-Lesches** [community/community#20983] [Projects and Issues]
Here's a sample list of the labels in one of my repositories: 
- backend
- bug
- enhancement
- networking
- documentation
- help wanted
- ....

In my mind, it is: 
- [Location in the code]: 
  - backend
  - frontend
  - networking
  - ...
- [Status of the issue]:
  - enhancement
  - bug
  - help wanted
  - ...

I get nested lists are a bit much, but at least being able to change the order in which labels appear in the dropdowns would make things more coherent. Thoughts?

**Comment by pitrou** [comment]
Since you didn't get an answer, I'd just mention that we solved this by prefixing each of the labels by their respective category. So in your case the labels could for example be named "Area: backend", "Area: frontend", "Type: enhancement", "Type: bug", etc.

**Reply by Levi-Lesches** [reply]
It's a very good workaround, I did that too in my repositories this year and I'm very happy with it. I'll still keep this issue up so users can decide whether `Type: bug` should appear before `Area: backend`.

**Comment by artumi-richard** [comment]
Another alternative is to use Projects to manage things. You can then add another field, one for status and one for location. But then that might not work well if you have something that has more than one location and/or status.

**Comment by bukzor** [comment]
The sad bit is that using the API I've gotten the labels to have the order I want, as seen when using the API, and the `gh` CLI uses that ordering, but the UI sorts them alphabetically regardless :( If we could just simply opt-out of the alphabetic label sorting, that would get us halfway there.

![image](https://github.com/community/community/assets/640328/1c77496a-0b3b-44dd-9e3c-1957d92299fb)
<img width="243" alt="image" src="https://github.com/community/community/assets/640328/dc22ac1f-1b6d-410d-b58d-3a581eb68016">
